app/controllers/podcast_episode_appearances_controller.rb

```diff
@@ -0,0 +1,114 @@
+class PodcastEpisodeAppearancesController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create edit update destroy]
+  before_action :set_episode, only: %i[new]
+  before_action :set_podcast_episode_appearance, only: %i[show edit update destroy]
+  before_action :set_episode_with_strong_params, only: %i[create update]
+  before_action :set_podcast_episode_appearances, only: %i[index create update destroy]
+
+  def index; end
+
+  def show; end
+
+  def new
+    @podcast_episode_appearance = @episode.podcast_episode_appearances.build
+    authorize @podcast_episode_appearance
+  end
+
+  def edit
+    authorize @podcast_episode_appearance
+  end
```

Comment:
What's the expected user experience here?

I'd imagine something like `"As the podcast owner I want to list the users that appeared on this episode"`

This makes me think we should have this functionality embedded in the `edit` page of the episode. It doesn't exist now, but I imagine the final UX to be:

- the owner goes to the podcast episode page
- clicks a "Manage episode" button that only the owner can see
- they are brought to `/episode-url/edit`
- they add or remove the owners